### PR TITLE
Improve the server endpoints section in the Info tab of Applications

### PR DESCRIPTION
**Pull request by savindi7**
### Purpose
- Remove webfinger endpoint
- Remove DCR endpoint (TODO - https://github.com/wso2/product-is/issues/18516)
- Show Session Iframe for SPA, TWA - OIDC, Standard based - OIDC applications only.

### Related Issues
- https://github.com/wso2/product-is/issues/18225

### Related PRs
- Related PR `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Comment by pavinduLakshan** [comment]
Add changeset

File: apps/console/src/features/applications/components/help-panel/oidc-configurations.tsx [hunk 1]
```diff
@@ -343,78 +350,36 @@ export const OIDCConfigurations: FunctionComponent<OIDCConfigurationsPropsInterf
...
-                                    />
-                                </Grid.Column>
-                            </Grid.Row>
+                            { (
+                                templateId === ApplicationManagementConstants.CUSTOM_APPLICATION_OIDC ||
+                                templateId === ApplicationManagementConstants.TEMPLATE_IDS.get("oidcWeb") ||
+                                templateId === ApplicationManagementConstants.TEMPLATE_IDS.get("spa")
+                            ) && (
+                                <Grid.Row columns={ 2 }>
+                                    <Grid.Column mobile={ 8 } tablet={ 8 } computer={ 6 }>
```
Comment (JayaShakthi97) [inline apps/console/src/features/applications/components/help-panel/oidc-configurations.tsx:359]:
```suggestion
                                    <Grid.Column mobile={ 8 } computer={ 6 }>
```

File: apps/console/src/features/applications/components/help-panel/oidc-configurations.tsx [hunk 2]
```diff
@@ -343,78 +350,36 @@ export const OIDCConfigurations: FunctionComponent<OIDCConfigurationsPropsInterf
...
+                                            className="left-icon"
+                                            verticalAlign="middle"
+                                            spaced="right"
+                                        />
+                                        <label data-testid={ `${ testId }-session-iframe-label` }>
+                                            { t("console:develop.features.applications.helpPanel.tabs.start.content." +
+                                                "oidcConfigurations.labels.sessionIframe") }
+                                        </label>
+                                    </Grid.Column>
+                                    <Grid.Column mobile={ 8 } tablet={ 8 } computer={ 10 }>
```
Comment (JayaShakthi97) [inline apps/console/src/features/applications/components/help-panel/oidc-configurations.tsx:375]:
```suggestion
                                    <Grid.Column mobile={ 8 } computer={ 10 }>
```

File: apps/console/src/features/applications/components/help-panel/oidc-configurations.tsx [hunk 3]
```diff
@@ -343,78 +350,36 @@ export const OIDCConfigurations: FunctionComponent<OIDCConfigurationsPropsInterf
...
+                                        <CopyInputField
+                                            value={ oidcConfigurations?.sessionIframeEndpoint  }
```
Comment (JayaShakthi97) [inline apps/console/src/features/applications/components/help-panel/oidc-configurations.tsx:377]:
```suggestion
                                            value={ oidcConfigurations?.sessionIframeEndpoint }
```

**Review by JayaShakthi97 — APPROVED** [review]

**Pull request merged**